### DISCUSSION TITLE
Invoke-FalconRTR

### DISCUSSION THREAD

**Discussion by Ainsley929** [CrowdStrike/psfalcon#94] [Q&A] [answered]
I'm getting a 50401:Exceeded maximum timeout error when trying to run a cloud file. I'm passing in around 400 hosts(error is on most hosts though some due queue offline) and I know the majority are online. Any idea how to resolve?

**Comment by bk-cs** [comment, marked as answer]
`50401: Exceeded maximum timeout` is due to the devices not completing the script in within the designated timeout window (default: 30 seconds). Try using the `-Timeout` parameter to increase it and see if that helps. If it doesn't, you might need to look into passing a script that starts a secondary process instead: https://github.com/CrowdStrike/psfalcon/wiki/Basic-Scripts#upload-and-execute-a-local-script-as-a-secondary-process
